### ISSUE TITLE
Add option to set bundled variations instead of using the default variation

### ISSUE THREAD

**Issue by ghost1227** [easydigitaldownloads/easy-digital-downloads#5066]
Copied from https://github.com/easydigitaldownloads/EDD-Software-Licensing/issues/788: 
---

## Overview:

We sell bundles on both our websites: WordImpress.com and Givewp.com - the bundles on wordimpress have the following variables: 
- 1 site
- 5 sites
- 15 sites

### Bundle screenshots:
![screenshot at oct 10 11-38-39](https://cloud.githubusercontent.com/assets/1571635/19247043/1ccf7792-8ede-11e6-879f-9d2bbad7885c.png)

![screen shot 2016-10-07 at 4 00 40 pm](https://cloud.githubusercontent.com/assets/1571635/19207807/b8b08c9c-8ca8-11e6-957c-427e183b3313.png)

### Download screenshot:
![screen shot 2016-10-07 at 4 01 02 pm](https://cloud.githubusercontent.com/assets/1571635/19207845/ff7c55ac-8ca8-11e6-93b9-eac2c33540ef.png)

## Problem

If I purchase a 5 site bundle from either site the licenses appear to be created properly in the back, but when I view the purchase confirmation and email receipt there are mixed license counts:

**These should all be "Up to 5 Sites":**
![screenshot at oct 07 15-59-40](https://cloud.githubusercontent.com/assets/1571635/19207928/a59254aa-8ca9-11e6-9a23-a92af8a046e1.png)

**These should all be "Up to 5 Sites":** 
![screenshot at oct 07 16-16-01](https://cloud.githubusercontent.com/assets/1571635/19207919/8da8f9d4-8ca9-11e6-986e-7e2b914a23c7.png)

**In the backend it appears fine:**

![screenshot at oct 07 16-16-49](https://cloud.githubusercontent.com/assets/1571635/19207914/7f5af9d6-8ca9-11e6-822a-951f2f28ea1c.png)

---

This is confusing to our customers and causing them to be frustrated when they purchase a certain license and discover we are giving them false information. A few have forwarded their emails and we've had to honor their requests to bump their license counts up for free.

**Comment by ghost1227** [comment]
I just realized that this issue was opened on the wrong repository. The user who opened it assumed it was an SL issue because of the licensing aspect, but the actual problem (as eventually confirmed by the user) actually has to do with how bundles are created. Bundles _should_ allow selecting which item in a variably priced product is included, but right now just enforce whatever the first option is.

**Comment by pippinsplugins** [comment]
This will be doable once #3175 is done (I think).

**Comment by michaelbeil** [comment]
I have seen this on two sites now. Both sites have a high customer base so we should look into getting this fixed sooner than later.

**Comment by michaelbeil** [comment]
Here is a reference to another ticket: https://secure.helpscout.net/conversation/263336215/44627/?folderId=725711

Also, @pippinsplugins, seeing that https://github.com/easydigitaldownloads/easy-digital-downloads/issues/3175 was done in 2.6.9, how should we milestone this?

**Comment by pippinsplugins** [comment]
Let's aim for 2.8.
